### ISSUE TITLE
Troublesome for Windows Telnet Client

### ISSUE THREAD

**Issue by spellbinder103** [glenndehaan/dark-chat#5]
have no issue with Linux Telnet Client but Windows Telnet Client. when using Windows Telnet Client connect to darkchat server, character is not showing correctly. what shall we do?

**Comment by glenndehaan** [comment]
Yeah it seems Windows is sending the newline char after every keypress, this is non-standard. I also see Windows is not recognizing the MOTD characters.

So for me this seems not fixable.
I did however try putty on windows in RAW mode. And that atleast solves the weird chars issue and fixes the newline issue. What it then doesn't fix is the window layout, so the output is all over the place

**Comment by spellbinder103** [comment]
Is fine if telnet have no color, however every times keypress will become trouble. Yes, Windows Telnet of Win 3.11

![image](https://user-images.githubusercontent.com/63778274/140878453-02953fca-885c-425f-b3d4-95aa8c218a8c.jpeg)

![image](https://user-images.githubusercontent.com/63778274/140878587-56697ac6-ff96-473a-ab6f-6ae3926e6e9a.jpeg)
